packages/xsnap/src/xsnap.js

```diff
@@ -458,10 +458,10 @@ export async function xsnap(options) {
         const handle = await fs.open(snapPath, 'w+');
         // @ts-expect-error 'close' event added in Node 15.4
         handle.on('close', () => {
-          fs.unlink(snapPath);
+          void fs.unlink(snapPath);
         });
         sourceStream = handle.createReadStream();
-        finished(output).finally(() => sourceStream.destroy());
+        void finished(output).finally(() => sourceStream.destroy());
```

Comment:
so this in an interesting case of unhandled promise. In this case I believe we want to silence any error that may occur.

```suggestion
        finished(output).finally(() => sourceStream.destroy()).catch(noop);
```